lib/foreman_maintain/cli/advanced_command.rb

```diff
@@ -8,6 +8,12 @@ class AdvancedCommand < Base
       subcommand 'prebuild-bash-completion',
                  'Prepare map of options and subcommands for Bash completion',
                  PrebuildBashCompletionCommand
+
+      if defined?(Procedures::ForemanTasks)
+        procedure = Procedures::ForemanTasks::Cleanup
+        klass = Class.new(Procedure::AbstractProcedureCommand) { params_to_options(procedure.params) }
+        subcommand(dashize(procedure.label), procedure.description, klass)
```

Comment:
Any specific reason behind having this as a subcommand  of `advanced`? 

The reason I am asking this because Foreman Maintain already has `foreman-maintain advanced procedure` command which allow users to run any available procedures.
Example - foreman-maintain advanced procedure run foreman-tasks-cleanup --help